libctru/source/services/soc/soc_common.h

```diff
@@ -10,6 +10,7 @@
 #include <3ds/services/soc.h>
 
 #define SYNC_ERROR ENODEV
+#define ADDR_STORAGE_LEN sizeof(struct sockaddr_storage)
```

Comment:
Maybe we should have a `_Static_assert(ADDR_STORAGE_LEN == 0x1c);`

Reply:
My idea was to not hardcode the size to any particular value (this PR is of the intent to revise the size of `sockaddr` after all, and I remember that the 3DS doesn’t have IPv6 support). It could be a helpful test though, I’ll see to add it if I have to make any changes.